components/index/SlideBackground.js

```diff
@@ -29,7 +29,7 @@ export default class SlideBackground extends React.Component {
     render() {
         return (
             <StyledSlideBackground
-                image={this.props.image}
+                image={this.props.image['large']}
```

Comment:
I have used media queries for now, because `object-fit: cover;` didn't behave exactly in the same way when animating as it did now. Maybe going to look into that later.